File: C++17/백준/Bronze/10178. 할로윈의 사탕/할로윈의 사탕.cc
#include <iostream>
using namespace std;

int main () {
    int ct;
    cin >> ct;
    
    
    
    for (int i = 0; i < ct; i++) {
        long long c, v;
        cin >> c >> v;
        
        long long p1 = c / v;
        long long p2 = c % v;
        
        cout << "You get " << p1 << " piece(s) and your dad gets " << p2 << " piece(s)." << endl;
    }
    
    return 0;
}

File: Java/백준/Bronze/10178. 할로윈의 사탕/할로윈의 사탕.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int tc = sc.nextInt();
        
        for (int i = 0; i < tc; i++) {
            long c = sc.nextLong();
            long v = sc.nextLong();
            
            long p1 = c / v;
            long p2 = c % v;
            
            System.out.println("You get " + p1 + " piece(s) and your dad gets " + p2 +" piece(s).");
        }
        sc.close();
    }
}